packages/amplify-appsync-simulator/src/velocity/util/elasticsearch-helper.ts

```diff
@@ -0,0 +1,194 @@
+import ElasticsearchUtils from "./elasticsearch-utils";
+
+class ElasticsearchHelper {
+  private static readonly ES_UTILS: ElasticsearchUtils = new ElasticsearchUtils();
+
+  private static readonly ERROR_FORMAT: string = "Could not construct an Elasticsearch Query DSL from {0} and {1}";
+
+
+  /**
+   * This method is used by the ModelTransformUtils.
+   *
+   * For Example, the following filter parameter:
+   *
+   * filter: {
+   *   title: {
+   *     eq: "hihihi",
+   *     wildcard: "h*i"
+   *   },
+   *   upvotes: {
+   *     gt: 5
+   *   }
+   * }
+   *
+   * will generate the following Elasticsearch Query DSL:
+   *
+   * {
+   *   "bool":{
+   *   "must":[
+   *     {
+   *     "range":{
+         "upvotes":{
+   *       "gt":5
+   *       }
+   *     }
+   *     },
+   *     {
+   *     "bool":{
+   *       "must":[
+   *       {
+   *         "term":{
+   *         "title":"hihihi"
+   *         }
+   *       },
+   *       {
+   *         "wildcard":{
+   *         "title":"h*i"
+   *         }
+   *       }
+   *       ]
+   *     }
+   *     }
+   *   ]
+   *   }
+   * }
+   *
+   * The default Operator is assumed to be `AND`.
+   *
+   * This will accept a FilterInput object, and return an Elasticsearch Query DSL Expression.
+   * This FilterInput object is defined by the appsync-model-transform code,
+   *  so this method is opinionated to follow this pattern.
+   *
+   * @param filterInput
+   *    The FilterInput object.
+   * @return
+   *    The Elasticsearch Query DSL
+   * 
+   */
+  public getQueryDSL(filterInput: any): any {
```

Comment:
Is there a stricter type that can be used instead of `any`? New code should not be using `any` unless absolutely necessary. Here and everywhere else.

Reply:
Determining the typings for this Appsync util methods is hard since it depends on user inputs to the search query.